ci/310-DEV_shapely20b1.yaml

```
@@ -0,0 +1,23 @@
+name: test
+channels:
+  - conda-forge
```

```suggestion
  - conda-forge
  - conda-forge/label/shapely_dev
```

dev version is on a different channel